packages/css/src/components/grid/grid.scss

```diff
@@ -24,48 +50,20 @@
   row-gap: initial;
 }
 
-.amsterdam-grid--gap-vertical--small {
-  row-gap: calc(var(--amsterdam-grid-gap) / 2);
-}
-
-.amsterdam-grid--gap-vertical--large {
-  row-gap: calc(var(--amsterdam-grid-gap) * 2);
-}
-
-.amsterdam-grid--padding-bottom--small {
-  padding-block-end: calc(var(--amsterdam-grid-gap) / 2);
-}
-
-.amsterdam-grid--padding-bottom--medium {
-  padding-block-end: var(--amsterdam-grid-gap);
-}
-
-.amsterdam-grid--padding-bottom--large {
-  padding-block-end: calc(var(--amsterdam-grid-gap) * 2);
-}
-
-.amsterdam-grid--padding-top--small {
-  padding-block-start: calc(var(--amsterdam-grid-gap) / 2);
-}
-
-.amsterdam-grid--padding-top--medium {
-  padding-block-start: var(--amsterdam-grid-gap);
-}
-
-.amsterdam-grid--padding-top--large {
-  padding-block-start: calc(var(--amsterdam-grid-gap) * 2);
-}
-
-.amsterdam-grid--padding-vertical--small {
-  padding-block: calc(var(--amsterdam-grid-gap) / 2);
-}
-
-.amsterdam-grid--padding-vertical--medium {
-  padding-block: var(--amsterdam-grid-gap);
+@each $class, $property in $amsterdam-grid-gap-classes {
```

Comment:
This has only one value, a loop should not be needed. Never in CSS IMO.

Reply:
It thought it was consistent and forward-looking, but it’s been removed due to the other comments.